### DISCUSSION TITLE
How do I add GitLab?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [super-productivity/super-productivity#2714] [Q&A] [answered]
It's not immediately obvious.

> ![image](https://github.com/johannesjo/super-productivity/assets/42837531/073a8a0c-d6eb-4e8b-9b08-cadee7cea8ac)

**Comment by roke-julian-lockhart** [comment, marked as answer]
> ![image](https://github.com/johannesjo/super-productivity/assets/42837531/ef826517-6982-467a-83b6-98ad84d3f444)

**Reply by Rohaq** [reply]
I'm having trouble figuring out what permissions I need to assign the Personal Access Token. It doesn't appear in any documentation I've found, and there aren't any suggestions in the UI - can you help? 😄

**Reply by roke-julian-lockhart** [reply]
> https://github.com/johannesjo/super-productivity/discussions/2714#discussioncomment-9490660

@Rohaq, because SP is a local application which I control, I merely provided all of the permissions possible. However, you're correct for not doing so - it's not ideal for security. I suggest that you create an issue requesting that the explicit permissions necessary be documented.

**Reply by Rohaq** [reply]
@RokeJulianLockhart Done, thanks! https://github.com/johannesjo/super-productivity/issues/3190
